plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -413,8 +413,10 @@ func (p *pusher) convertEvent(e logs.LogEvent) *cloudwatchlogs.InputLogEvent {
 			// a valid timestamp and use the last valid timestamp for new entries that does
 			// not have a timestamp.
 			t = p.lastValidTime
+			p.Log.Debugf("Unable to parse valid timestamp, using last valid timestamp %v: for log group %v: ", p.lastValidTime, p.Group)
```

Comment:
We don't want to log this every time. There are valid scenarios where this path would be accessed. This log line would be misleading.

Reply:
Yeah I will add a interval check